inginious/common/tests/test_course_user_settings.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=redefined-outer-name
+#
+# This file is part of INGInious. See the LICENSE and the COPYRIGHTS files for
+# more information about the licensing of this file.
+
+import pytest
+
+from inginious.common.course_user_setting import CourseUserSetting
+
+
+@pytest.fixture()
+def init():
+    af = CourseUserSetting("test", "a description", 1)
+    yield af
+
+
+class TestCourseUserSetting(object):
+    """Test for course user settings class"""
+
+    def test_course_user_settings_init(self, init):
+        af = init
+        assert af is not None
+        af = CourseUserSetting(0, "a description", 1)
+        try:
+            af = CourseUserSetting("fail", "a description", 99)
+        except Exception:
+            assert True
```

Comment:
`assert True` will always pass. If the code fails to throw an exception, the test itself will pass. You may want to `assert False` right after the constructor instead.